src/test/java/com/ixortalk/organization/api/rest/OrganizationRestResource_Add_IntegrationAndRestDocTest.java

```diff
@@ -111,6 +110,7 @@ public void asAdmin() throws JsonProcessingException {
 
         verify(auth0Roles).addRole(EXPECTED_GENERATED_ROLE_NAME_FOR_MY_TEST_ORGANIZATION);
         verify(auth0Roles, never()).assignRolesToUser(anyString(), anySet());
+        assertThat(restResourcesTransactionalHelper.getUsers("My Organization").size()).isEqualTo(0);
```

Comment:
Why querying by name?  Isn't querying by `myTestOrganization.getId()` nicer?  Or extract the name to a constant.

Reply:
> Why querying by name? Isn't querying by `myTestOrganization.getId()` nicer? Or extract the name to a constant.

The id is not directly available in that pojo. The id needs to be parsed from the response body, which is imho a hassle,  so I decided to add another helper method. I don't see the problem though?

Reply:
Oh yes, off course, got confused by the `@Before` but it's obviously not yet persisted there.